chatter/package.json

```diff
@@ -3,19 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "engines": {
-    "node" : ">= 15.0.0"
+    "node": ">= 15.0.0"
   },
   "dependencies": {
     "@mdi/font": "^5.9.55",
     "@react-hook/mouse-position": "^4.1.0",
     "animate.css": "^4.1.1",
     "classnames": "^2.2.6",
-    "sass": "^1.43.4",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-scripts": "4.0.3",
+    "rxjs": "^7.8.1",
+    "sass": "^1.43.4",
     "socket.io-client": "^3.1.3",
-    "use-sound": "^2.0.1",
+    "use-sound": "^4.0.1",
```

Comment:
Upgrade `use-sound` as the `2.0.1` was buggy (https://github.com/joshwcomeau/use-sound/issues/42) that cause unwanted re-rendering.